dbt/README.md

```diff
@@ -494,63 +468,141 @@ We use the following pattern to determine where to define each column descriptio
-      expression produces the correct output for a given input string
-
-dbt tests are data tests by default, although a dedicated unit testing syntax
-[is coming soon](https://docs.getdbt.com/docs/build/unit-tests). Until unit
-tests are natively supported, however, we do not have a way of implementing them.
-We plan to change this once unit testing is released, but for now, make sure that
-any new tests you write are data tests and not unit tests.
-
-### Adding data tests
+      expression in a view produces the correct output for a given input string
+3. [**QC reports**](#qc-reports) check for suspicious cases that _might_ indicate a problem with
+   our data, but that can't be confirmed automatically
+    * For example: Query for all parcels whose market value increased by
+      more than $500k in the last year
+  
+The following sections describe how to add and run each of these types of products.
+
+### Data tests
+
+We implement data tests using [dbt tests](https://docs.getdbt.com/docs/build/tests)
+to check that hard-and-fast assumptions about our raw data are correct. We prefer adding tests
+inline in `schema.yml` config files using [generic
+tests](https://docs.getdbt.com/best-practices/writing-custom-generic-tests),
+rather than [singular
+tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests).
 
-There are two types of data tests that we support:
+There are two subtypes of data tests that we support:
 
 1. **QC tests** confirm our assumptions about iasWorld data and are run at
    scheduled intervals to confirm that iasWorld data meets spec
 2. **Non-QC tests** confirm all other assumptions about data sources outside
    of iasWorld, and are run in an ad hoc fashion depending on the needs of
    the transformations that sit on top of the raw data
 
-#### Adding QC tests
+**QC tests** can be run using the [`dbt test`
+command](https://docs.getdbt.com/reference/commands/test) and their output can be
+transformed for analysis using the [`transform_dbt_test_results`
+script](https://github.com/ccao-data/data-architecture/blob/master/dbt/scripts/transform_dbt_test_results.py).
+This script reads the metadata for the most recent `dbt test` run and outputs a number of
+different artifacts with information about the tests:
+
+* An Excel workbook with detailed information on each failure to aid in resolving
+  data problems
+* Parquet files representing metadata tables that can be uploaded to S3 for aggregate
+  analysis
+
+**Non-QC tests** are much less common than QC tests in our test suite. We don't currently
+have a great process for handling them, so there is not much to document yet. If you
+are being asked to add a test that appears to be a non-QC test, meaning that it
+does not confirm our assumptions about iasWorld data, double check with the person who
+assigned the test to you and ask them when and how the test should be run so that its
+attributes can be set accordingly.
```

Comment:
I wonder if it's even worth mentioning this distinction, given that we don't really support non-QC tests anymore? And perhaps it would be more accurate to describe the distinction here as being between iasWorld QC tests and non-iasWorld QC tests? I'm a little unsure, curious what you think!

Reply:
AFAIK the non-QC tests are only run in the context of dbt builds. I think that's useful and worth keeping around, as there may be non-unit, non-QC tests that we want to perform related to table relations between stuff not in iasWorld. 

I agree this is pretty confusing though. Building off my earlier thought, what if renamed things as:

- **QC tests** --> **Data Integrity (DI) tests**
- **Non-QC tests** --> **Integration tests**
- **Unit tests** (unchanged)
- **QC reports** (unchanged)